build.csx

```diff
@@ -0,0 +1,217 @@
+
+targets.Add("logsDirectory", () => Directory.CreateDirectory(logsDirectory));
+
+targets.Add("testsDirectory", () => Directory.CreateDirectory(testsDirectory));
+
+targets.Add("build", DependsOn("clean", "outputDirectory", "restore"), () => RunMsBuild("Build"));
+
+targets.Add("clean", DependsOn("logsDirectory"), () => RunMsBuild("Clean"));
+
+targets.Add(
+    "restore",
+    () =>
+    {
+        Cmd(nuget, $"restore {solution} -PackagesDirectory {packagesDir}");
+        foreach (var projectDir in netstdUnitTestDirectories.Concat(netstdIntegrationTestDirectories).Concat(netstdSpecDirectories))
+        {
+            Cmd(projectDir, "dotnet", "restore");
+        }
+    });
+
+targets.Add(
+    "unit",
+    DependsOn("build", "testsDirectory"),
+    () =>
+    {
+        RunTests(unitTests);
+        RunDotNetTests(netstdUnitTestDirectories);
+    });
+
+targets.Add(
+    "integ",
+    DependsOn("build", "testsDirectory"),
+    () =>
+    {
+        RunTests(integrationTests);
+        RunDotNetTests(netstdIntegrationTestDirectories);
+    });
+
+targets.Add(
+    "spec",
+    DependsOn("build", "testsDirectory"),
+    () =>
+    {
+        RunTests(specs);
+        RunDotNetTests(netstdSpecDirectories);
+    });
+
+targets.Add(
+    "approve",
+    DependsOn("build", "testsDirectory"),
+    () => RunTests(approvalTests));
+
+targets.Add(
+    "pack",
+    DependsOn("build", "outputDirectory"),
+    () =>
+    {
+        var version = GetVersion();
+        foreach (var nuspec in nuspecs)
+        {
+            Cmd(nuget, $"pack {nuspec} -Version {version} -OutputDirectory {outputDirectory} -NoPackageAnalysis");
+        }
+    });
+
+targets.Add(
+    "gitlink",
+    DependsOn("build"), () => Cmd(gitlink, $". -f {solution} -u {repoUrl} -include " + string.Join(",", gitlinks)));
+
+Run(Args, targets);
+
+// helpers
+public static void Cmd(string fileName, string args)
+{
+    Cmd(".", fileName, args);
+}
+
+public static void Cmd(string workingDirectory, string fileName, string args)
+{
+    using (var process = new Process())
+    {
+        process.StartInfo = new ProcessStartInfo
+        {
+            FileName = $"\"{fileName}\"",
+            Arguments = args,
+            WorkingDirectory = workingDirectory,
+            UseShellExecute = false,
+        };
+
+        var workingDirMessage = workingDirectory == "." ? "" : $" in '{process.StartInfo.WorkingDirectory}'";
+        Console.WriteLine($"Running '{process.StartInfo.FileName} {process.StartInfo.Arguments}'{workingDirMessage}...");
+        process.Start();
+        process.WaitForExit();
+        if (process.ExitCode != 0)
+        {
+            throw new InvalidOperationException($"The command exited with code {process.ExitCode}.");
+        }
+    }
+}
+
+public static void RunMsBuild(string target)
+{
+    var packagesDirOption = string.IsNullOrEmpty(packagesDir) ? "" : $"/p:NuGetPackagesDirectory={packagesDir}";

```

Comment:
If `packagesDir` were to become `packagesDirectory` then `packagesDirOption` ought to change similarly.